src/display/editor/editor.js

```diff
@@ -812,6 +818,7 @@ class AnnotationEditor {
     }
     const altText = (this.#altTextButton = document.createElement("span"));
     altText.className = "altText";
+    altText.setAttribute("role", "button");
```

Comment:
Possibly a dumb idea on my part, by would it make (any) sense to have the altText DOM-element be an actual `<button>` instead of "faking" that here?
Or would that perhaps make styling more challenging?